irohad/main/application.hpp

```diff
@@ -370,6 +372,33 @@ class Irohad {
   logger::LoggerManagerTreePtr log_manager_;  ///< application root log manager
 
   logger::LoggerPtr log_;  ///< log for local messages
+
+  std::shared_ptr<iroha::subscription::SubscriberImpl<
+      iroha::EventTypes,
+      iroha::SubscriptionDispatcher,
+      bool,
+      iroha::synchronizer::SynchronizationEvent>>
+      syncSubscription;
+
+  using OnProposalSubscription =
+      iroha::subscription::SubscriberImpl<iroha::EventTypes,
+                                          iroha::SubscriptionDispatcher,
```

Comment:
I did not see any SubscriberImpl with tparams other than <iroha::EventTypes,iroha::SubscriptionDispatcher,...>

Lets `using` it as BasicSubsciber<Receiver,NotificationArgs>

Reply:
https://imgur.com/3l4KntA.png